Adds bug, feature and task issue templates (#20)
IcaliaLabs/dojo: dojo-templates/issue_templates/user-story.md

---
name: User Story
about: Use this to describe a new feature or user story for the project

---

> As < user type > I can < description of the feature > in order to < goal >

Reviewer: ```suggestion
As a/an < user type > I can < description of the feature > in order to < goal >
```

## Context

You can add a bit more context for the feature, if there is already a plugin implementing this, or some helpful hints.

## TO-DO

- [ ] A task for the user story
- [ ] A task for the user story
- [ ] A task for the user story

## Acceptance criteria

- Acceptance criteria 1
- Acceptance criteria 2
- Acceptance criteria 3

> ## Files that might be modified or being affected

Reviewer: ```suggestion
## Files that might need modification or are being affected
```

- `app/models/user.rb`
- `app/views/users/profile.html.erb`

> ## Relationships

Reviewer: ```suggestion
## Related Issues and Pull Requests
```

> A list of related issued if apply or link to resources which the developer may need, such as the invision url for the design.

Reviewer: ```suggestion
A list of related issues if any apply or link to resources which the developer may need, such as the Invision url for a design reference.
```
